cypress/integration/personal-information.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { PersonalInformation } from  "../page/index"
+
+describe("Fill a form", () => {
+
+    let personalInformation: PersonalInformation;
+
+    before(() => {
+
+        personalInformation = new PersonalInformation;
+    })
+
+    it("then the form must be fill", () => {
+
+
+        personalInformation.visitFormPage()
+        personalInformation.InsertName("Holmes")
+        personalInformation.InsertLastName("Salazar")
+        personalInformation.InsertEmail("hibarras@email.com")
+        personalInformation.SelectGender("Male")
+        personalInformation.InsertMobile("3656589156")
+        personalInformation.InsertHobbies(["Reading", "Music"])
+        personalInformation.InsertAddress("Av siempreViva # 123")
+        personalInformation.Submit()
```

Comment:
the idea is to have just one method called `fillForm`, which receives an object with the data to be typed in the form. For that, you will need to create a class or interface to represent the personal information data.

After that, you should create assertions to verify that the page that shows the data after the submit button is clicked contains the same data you entered on the `fillForm` method.

Reply:
Thank you, I will work on that

Reply:
I already added method, class, and object. I'm still working on the modal validation. 
Please let know  me what do you think about this

Thank you